src/App.test.tsx

```diff
@@ -7,6 +7,15 @@ import { shallow } from 'enzyme';
 import useHeight from './hooks/useHeight/useHeight';
 import useRoomState from './hooks/useRoomState/useRoomState';
 
+jest.mock('swiper/react/swiper-react.js', () => ({
+  Swiper: jest.fn(),
+  SwiperSlide: jest.fn(),
+}));
+
+jest.mock('swiper', () => ({
+  Pagination: jest.fn(),
+}));
+
```

Comment:
This is clearly needed to make the tests pass, but I can't understand why 😭 . Everything is shallow rendered. Do you have any ideas?

Reply:
Jest is failing due to a syntax error: `SyntaxError: Cannot use import statement outside a module`. Sent you the screenshot in slack.. not entirely sure why mocking the file makes it pass though 🤔